integrations/phoneinfoga/compose-tests.yml

```diff
@@ -0,0 +1,13 @@
+version: '3.7'
+
+services:
+    phoneinfoga:
+      container_name: phoneinfoga
+      restart: unless-stopped
+      image: sundowndev/phoneinfoga:latest
+      espose:
+        - "5000"
+      env_file:
+      - env_file_integrations
+      environment:
+      - GIN_MODE=release
```

Comment:
copy/pastes are not required because the `compose-tests.yml` is an override of the `compose.yml`. Please use other docker analyzers as a reference

Reply:
so should this file even contain anything as we are using an external service; would it make sense to have an additional dockerfile that clones the repo and builds it for the `compose-tests.yml`?

Reply:
you are right, we could basically leave this almost empty.

Reply:
should I just mention `version: 3.8` in the test yml and rest empty?